src/CMakeLists.txt

```diff
@@ -113,31 +114,24 @@ if(MANIFOLD_EXPORT)
 endif()
 target_compile_features(manifold PUBLIC cxx_std_17)
 if(MANIFOLD_PAR)
-  target_compile_options(manifold INTERFACE -DMANIFOLD_PAR)
+  target_compile_options(manifold PUBLIC -DMANIFOLD_PAR)
```

Comment:
This is an interesting one, `INTERFACE` will make the compile option only available to external code linking with our library, so it is not actually using parallelization. Changing to public fixes that.

Reply:
and this makes me thinking if we should *always* set these to some non-zero values and assert the thing should be non-zero in our code, to avoid this sort of issue (and potential typos, which we had before)

Reply:
Urk.  That's my mistake - I was misunderstanding the PUBLIC vs. INTERFACE settings.  I thought PRIVATE was just for the target, PUBLIC was for all other targets in the current CMake project linking that target, and INTERFACE meant everything... I see now that is incorrect, apologies.

For future reference (mostly for me) here's the link to CMake's docs on those three keywords:
https://cmake.org/cmake/help/latest/manual/cmake-buildsystem.7.html#target-command-scope

Reply:
@pca006132 Were you thinking something like this for asserting non-zero for parallel values?  https://github.com/starseeker/manifold/commit/c94e8a20b1205b0cfc725d72c9d568a011d07e62

Reply:
@starseeker maybe a bit simpler, 1 for true and -1 for false. This will work for other macros as well.

Reply:
@pca006132 Updated.  Pull request up: https://github.com/elalish/manifold/pull/980